docs/README.md

```diff
@@ -64,6 +64,55 @@ _Example of a wishlist page._
 
 > ℹ️ _The new route called `/wishlist`, responsible for creating the Wishlist custom page that displays wishlisted product items, already contains a default template, it is already rendered under the My Account menu and no further actions are required from you. However, you can **customize the Wishlist page, overwriting the template by creating a brand new one as you wish**. To do so, check the **Advanced configurations** section below._
 
+## Uages
+
+There are couple URLs to read, search and modify data for the app:
+
+To read the schema of the wishlist app: 
+`````
+curl --request GET \ 
+     --url 'https://{environment}--{accountName}.myvtex.com/api/dataentities/wishlist/schemas/' \
+     --header 'VtexIdClientAutCookie: {authToken}' \
+`````
+
+To GET all the wishlist data: 
+`````
+curl --request GET \
+     --url 'https://{environment}--{accountName}.myvtex.com/_v/wishlist/export-lists' \
```

Comment:
I think the format https://{{accountName}}.vtexcommercestable.com.br/_v/wishlist/export-lists would work here, but I'm not completely sure.  https://{accountName}.myvtex.com/_v/wishlist/export-lists or https://{workspace}--{accountName}.myvtex.com/_v/wishlist/export-lists should be fine.